main.go

```diff
@@ -182,7 +184,16 @@ func generateCharts(c *cli.Context) {
 			// New chart was introduced
 			modifiedAssetPath = fmt.Sprintf("%s/*", filepath.Join(path.RepositoryAssetsDir, splitPath[1], splitPath[2]))
 		default:
-			// Existing chart was modified
+			if strings.HasSuffix(splitPath[2], "-crd") {
+				// e.g. p = charts/rancher-monitoring/rancher-monitoring-crd/100.1.0+up19.0.3/files/crd-manifest.tgz
+				chartKey := fmt.Sprintf("%s-%s-%s", splitPath[1], splitPath[2], splitPath[3])
+				if strings.HasSuffix(p, ".tgz") {
+					modifiedCrdTgzPaths[chartKey] = p
+					continue
+				}
```

Comment:
As discussed offline, let's see if we can eliminate this second map in favor of just having the key be something like:
```go
chartKey := filepath.Join(splitPath[0], splitPath[1], splitPath[2]), splitPath[3], "files", "crd-manifest.tgz")
```

Reply:
This is fixed. 
I also move the tgz filename to a const variable.